packages/manager/apps/hub/src/app.module.js

```diff
@@ -119,6 +121,11 @@ angular
         unregisterHook();
       });
     },
+  )
+  .config(
+    /* @ngInject */ (ovhFeatureFlippingProvider) => {
+      ovhFeatureFlippingProvider.setApplicationName('hub');
```

Comment:
Looks like a duplicate of a Pending Pull request.
see: https://github.com/ovh/manager/pull/4124/files#diff-17953c7ed48e08a390075d0a4ab263796472f2d1e6c2a1d167a7eef2fb72b85aR71

Maybe we can safely merge #4124, Isn't it?
Do you agree with that?

Reply:
Yep same thing. 
If #4124 is merged I will rebase this one.

Reply:
Thank you!

#4124 has been merged in `develop`.

Reply:
Thanks I will rebase.

Reply:
Rebased.